contracts/keep3r-liquidity-manager/Keep3rLiquidityManagerUserLiquidityHandler.sol

```diff
@@ -40,13 +56,42 @@ abstract contract Keep3rLiquidityManagerUserLiquidityHandler is Keep3rLiquidityM
   using SafeMath for uint256;
   using SafeERC20 for IERC20;
 
+  uint256 public constant PRECISION = 1_000;
+  // liquidity fee
+  uint256 public override liquidityFee;
+  // feeReceiver address
+  address public override feeReceiver;
+  // lp => minAmount
+  mapping(address => uint256) public override liquidityMinAmount;
   // lp => amount (helps safely collect extra dust)
   mapping(address => uint256) public override liquidityTotalAmount;
   // user => lp => amount
   mapping(address => mapping(address => uint256)) public override userLiquidityTotalAmount;
   // user => lp => amount
   mapping(address => mapping(address => uint256)) public override userLiquidityIdleAmount;
 
+  constructor(address _keep3rV1) public Keep3rLiquidityManagerParameters(_keep3rV1) {}
+
+  // governor
+  function _setLiquidityFee(uint256 _liquidityFee) internal {
+    // TODO better revert messages
+    require(_liquidityFee <= PRECISION, 'fee-exceeds-100-percent');
+    liquidityFee = _liquidityFee;
+    emit LiquidityFee(_liquidityFee);
+  }
+
+  function _setFeeReceiver(address _feeReceiver) internal {
```

Comment:
missing event emitter

Reply:
no needed, since it's not something outsiders need to care about